src/pages/favorites.module.scss

```diff
@@ -0,0 +1,42 @@
+@use "../sass/mixins.scss" as *;
+
+.favorites {
+  display: grid;
+  grid-template-columns: repeat(1, 1fr);
+  gap: 1rem;
+  min-height: 100vh;
```

Comment:
If it cause the scroll bar when it's not needed then it's a not acceptable solution